docs/howto/ipdb_how_to_use_it_with_nornir.rst

```diff
@@ -0,0 +1,74 @@
+ipdb: How to use it with Nornir
+===============================
+
+Python comes with a debuger called pdb. The python debugger is very powerful and really facilitates troubleshooting code. You can find the official documentation `here <https://docs.python.org/3.6/library/pdb.html>`_. `ipdb <https://github.com/gotcha/ipdb>`_ builds on top of ipython and pdb to give users a more interactive experience.
+
+This how to isn't very extensive, it's just a very quick and dirty demo to show some of it's capabilities and how to use it with nornir. It doesn't assume previous knowledge but doesn't spend too much on explanations either so it expects users to give enough material for further investigation.
+
+Installing ipdb
+---------------
+
+First, you will need to install `ipython <http://ipython.org/>`_, follow the official guide to do so, then you will need to install ``ipdb``. You can install the latter via pip::
+
+   pip install ipdb
+
+Finding and fixing a bug with ipdb
+----------------------------------
+
+For the sake of the demo I have written some code that contains a bug. The code is not important but what it does is:
+
+#. Read a yaml file containing a bunch of users we want to have configured on our devices
+#. Connect to a couple of network devices and get the users configured
+#. Check with users are already configured, which ones we want and compute two lists; a list with the users we need to configure and another list with the ones we need to remove.
```

Comment:
`Check with users are already configured,` should read `Check which users are already configured,`